devel/ragel/Makefile

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (C) 2023 OpenWrt.org
```

Comment:
please either assign the copyright to yourself or simply remove this section.

Reply:
Removed, thanks for the feedback.